src/Tools/AnalyzerRunner/IncrementalAnalyzerRunner.cs

```diff
@@ -61,17 +59,6 @@ public async Task RunAsync(CancellationToken cancellationToken)
                     throw new InvalidOperationException("Benchmark is not configured to use persistent storage.");
                 }
             }
-
-            var incrementalAnalyzerProviders = exportProvider.GetExports<IIncrementalAnalyzerProvider, IncrementalAnalyzerProviderMetadata>();
-            foreach (var incrementalAnalyzerName in _options.IncrementalAnalyzerNames)
```

Comment:
I believe we will also need to make changes to the supported command line arguments for AnalyzerRunner to no longer support `IncrementalAnalyzerNames` option. FYI @sharwell